### ISSUE TITLE
TRECCollection is difficult to use from Python

### ISSUE THREAD

**Issue by cmacdonald** [terrier-org/terrier-core#139]

**Comment by cmacdonald** [comment]
```python
def createCollection(files):
    default_properties = {
            "TrecDocTags.doctag": "DOC",
            "TrecDocTags.idtag": "DOCNO",
            "TrecDocTags.skip": "DOCHDR",
            "TrecDocTags.casesensitive": "false",
            "trec.collection.class": "TRECCollection",
    }
    for (k,v) in default_properties.items():
        pt.ApplicationSetup.setProperty(k, v)
    asList = pt.autoclass("java.util.Arrays").asList(*files)
    cls_string = pt.autoclass("java.lang.String")._class
    cls_list = pt.autoclass("java.util.List")._class
    return pt.autoclass("org.terrier.indexing.CollectionFactory").loadCollections(
        ["TRECCollection"],
        [cls_list, cls_string, cls_string, cls_string],
        [asList, pt.autoclass("org.terrier.utility.TagSet").TREC_DOC_TAGS, "", ""])
```

Minimal example to open TRECCollection on a list of filenames. CollectionFactory also not useful!

**Comment by cmacdonald** [comment]
Addressed for 5.4
